### ISSUE TITLE
Sumo Dvent

### ISSUE THREAD

**Issue by PsychoMcPeYTDEV** [Atm85/EventsPro#2]
is this a sumo event that tps players to sumo 1v1, one by one? like a tournament, and does still works?

**Comment by Atm85** [comment]
> is this a sumo event that tps players to sumo 1v1, one by one? like a tournament, and does still works?

Yes, it is a Suno 1v1 plugin and tp's players to an arena, however, its been awhile since iv worked on this so idk if it still works...
It does have a compatible api version so you can try and see!

**Comment by PsychoMcPeYTDEV** [comment]
how can i download? i'm on ios, and i can't use desktop mode, can you give me the zip?
